Adding Lora firmware update instructions (#111)
arduino/arduino-pro-content: content/tutorials/portenta-h7/vs-ard-ttn/content.md

- USB C cable (either USB A to USB C or USB C to USB C)
- An [account](https://account.thethingsnetwork.org/users/login) with The Things Network

> ### Updating LoRa Module Firmware

Reviewer: ```suggestion
### Updating the LoRa® Module Firmware
```

> To be able to use the LoRa functionality, we need to first update the firmware on the LoRa modem, this is simply done through Arduino IDE and an example included in the MKRWAN library.

Reviewer: ```suggestion
To be able to use the LoRa® functionality, we need to first update the firmware on the LoRa® modem. This can be done through Arduino IDE by running a sketch included in the examples from the MKRWAN library.
```

> 1. Connect the Portenta and Vision shield to your computer and open the Arduino IDE. 

Reviewer: ```suggestion
1. Connect the Portenta H7 and the Portenta Vision Shield to your computer and open the Arduino IDE. 
```

> 2. Install/update the **MKRWAN** library from Arduino IDE menu  _Tools > Manage Libraries_ type in "MKRWAN" to find the library and click 'Install' or 'Update' if necessary. This library provides all the APIS to communicate with LoRa and LoRaWAN networks.

Reviewer: ```suggestion
2. Install/update the **MKRWAN** library from Arduino IDE menu  **Tools > Manage Libraries**. Type "MKRWAN" to find the library and click 'Install' or 'Update' if necessary. This library provides all the APIs to communicate with LoRa® and LoRaWAN® networks.
```

> 3. Upload the **MKRWANFWUpdate_standalone** sketch from Arduino IDE top menu _File > Examples > MKRWAN_.

Reviewer: ```suggestion
3. Open the **MKRWANFWUpdate_standalone** sketch from the Arduino IDE  menu: **File > Examples > MKRWAN**.

4. Upload the sketch.

```

> ![Finding the add application button](assets/vs_ard_ttn_standalone.png)

Reviewer: Image caption not correct

> 4. Open the serial monitor and wait for confirmation.

Reviewer: ```suggestion
5. Open the serial monitor and wait for the update to be confirmed.
```

> ![Finding the add application button](assets/vs_ard_ttn_serial.png)

Reviewer: Image caption not correct
